programs/redstone-sol/src/state.rs

```diff
@@ -0,0 +1,37 @@
+use anchor_lang::prelude::*;
+
+pub type SignerAddress = [u8; 20];
+pub type FeedId = [u8; 32];
+
+#[account]
+#[derive(Default)]
+pub struct PriceData {
+    pub feed_id: FeedId,
+    pub value: u128,
```

Comment:
Same as below, we'd like to have here `U256`. But, as I've checked, it's not serializable by default ;(

Reply:
works all good! 1ff23264ddfb53005201a472edfebdca996243d5, thanks to borsh 😸